src/LayoutManager/ActiveSelectionLayoutManager.ts

```diff
@@ -0,0 +1,27 @@
+import { LayoutManager, buildStandardEvents } from './LayoutManager';
+import type { RegistrationContext, StrictLayoutContext } from './types';
+import type { FabricObject } from '../shapes/Object/FabricObject';
+
+/**
+ * The ActiveSelectionLayoutManager is exactly as the LayoutManager
+ * But what it does differently is that it subscribes the activeSelection
+ * modification events to modify the layout of the original group of the
+ * children. This is done by swapping target and object in the subscribeTargets
```

Comment:
> This is done by swapping target and object in the subscribeTargets

IMO this sentence is more confusing than helpful, especially for the dev

Reply:
What would you write?